app/presenters/finder_breadcrumbs_presenter.rb

```diff
@@ -0,0 +1,23 @@
+class FinderBreadcrumbsPresenter
+  def initialize(parent_content_item, finder_content_item)
+    @parent_content_item = parent_content_item
+    @finder_name = finder_content_item.dig("title")
+  end
+
+  def breadcrumbs
+    crumbs = [{ title: "Home", url: "/" }]
+
+    if @parent_content_item.dig("document_type") == "organisation"
+      crumbs << { title: 'Organisations', url: '/government/organisations' }
+      if @parent_content_item.dig("title").present?
+        crumbs << { title: @parent_content_item.dig("title"), url: @parent_content_item.dig("base_path") }
+      end
+    end
+
+    if @finder_name.present?
+      crumbs << { title: @finder_name }
```

Comment:
I just spotted that we need to put `is_current_page: true` in this hash: https://govuk-publishing-components.herokuapp.com/component-guide/breadcrumbs/highlight_current_page .  

It changes the `aria-current` attribute on that breadcrumb item.